package.json

```diff
@@ -88,6 +92,7 @@
     "typescript": "^3.3.3",
     "untildify": "^4.0.0",
     "unzipper": "^0.10.1",
+    "uuid": "^3.3.3",
```

Comment:
Think we already have a UUID lib

Reply:
Nice.  Let me use that one.

Reply:
Was it pulled in as a transitive dependency?

Reply:
Switched to shortid